tests/improved-sanity-test/main.yml

```diff
@@ -185,6 +216,23 @@
       tags:
         - semanage_add_verify
 
+    # Check layered package is still installed
+    - role: rpm_ostree_install_verify
+      package: "{{ g_pkg }}"
+      tags:
+        - rpm_ostree_install_verify
+
+    # Check admin unlock overlayfs is no longer there after upgrade
+    - role: overlayfs_verify_missing
+      tags:
+        - overlayfs_verify_missing
+
+    # Verify admin unlocked package still installed
```

Comment:
Seems like it should read:

`# Verify admin unlocked package is removed after upgrade`